### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Towards https://github.com/giantswarm/cli/issues/509

### CODE REVIEW

**Comment by JosephSalisbury** [comment]
This fixes 3 out of 4 of the failing int-tests, with these three being two messaging changes, and disabling one test

I'm leaving the fourth test (the unresolved variable message test) for a separate PR, in the interest of keeping this pr simple

ping @giantswarm/team-features

**Comment by marians** [comment]
Looking good, but I can't test right now

File: int-tests/006-api-endpoint-arg.t [hunk 1]
```diff
@@ -2,5 +2,7 @@ Ping:
 
   $ swarm --debug --verbose --api-endpoint=http://www.__giantswarm.io/notthere ping
   DEBUG: request ID '[a-z0-9]+' (re)
-  FAILED: Get http://www.__giantswarm.io/notthere/v1/ping: dial tcp: lookup www.__giantswarm.io.* (re)
+  DEBUG: GET http://www.__giantswarm.io/notthere/v1/ping
+  ERROR: << Get http://www.__giantswarm.io/notthere/v1/ping: dial tcp: lookup www.__giantswarm.io on 194.8.194.60:53: no such host
+  FAILED: Get http://www.__giantswarm.io/notthere/v1/ping: dial tcp: lookup www.__giantswarm.io on 194.8.194.60:53: no such host
```
Comment (zyndiecate) [inline int-tests/006-api-endpoint-arg.t]: I think this is not correct. The IP will differ from time to time. That is why there was this regular expression.

**Comment by JosephSalisbury** [comment]
RFR

File: int-tests/006-api-endpoint-arg.t [hunk 2]
```diff
@@ -2,5 +2,7 @@ Ping:
 
   $ swarm --debug --verbose --api-endpoint=http://www.__giantswarm.io/notthere ping
   DEBUG: request ID '[a-z0-9]+' (re)
+  DEBUG: GET http://www.__giantswarm.io/notthere/v1/ping
+  ERROR: << Get http://www.__giantswarm.io/notthere/v1/ping: dial tcp: lookup www.__giantswarm.io.* (re)
```
Comment (zyndiecate) [inline int-tests/006-api-endpoint-arg.t]: Ah, now I get it. This is because we fixed the debug output to always show the URL causing the failure.

**Comment by JosephSalisbury** [comment]
Works for @marians , merging

**Comment by zyndiecate** [comment]
LGTM

**Pull request merged**